backend/api/Areas/Reports/Controllers/LeaseController.cs

```diff
@@ -151,6 +153,23 @@ public IActionResult ExportAggregatedLeases(int fiscalYearStart)
         }
 
         #endregion
+
+        /// <summary>
+        /// Create duplicate lease rows for every unique property lease, tenant, and term.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <param name="all"></param>
+        /// <returns></returns>
+        public IEnumerable<LeaseModel> GetCrossJoinLeases(Lease.Models.Search.LeaseFilterModel filter, bool all = false)
+        {
+            filter.Quantity = all ? _pimsRepository.Lease.Count() : filter.Quantity;
+            var page = _pimsRepository.Lease.GetPage((LeaseFilter)filter);
```

Comment:
Instead of casting we should use the constructor or Automapper for the type conversion

Reply:
This is an existing pattern where an explicit casting operator is added between the model and dal versions of the filter. If we don't like that we should probably log some tech debt to fix all of the other instances of this.